docs/guides/transactions.md

```diff
@@ -6,6 +6,11 @@ A transaction is basically a context manager that collects all file uploads, def
 They are an "all or nothing" proposition: If an error occurs during the transaction, none of the queued files are uploaded.
 For more information on fsspec transactions, see the official [documentation](https://filesystem-spec.readthedocs.io/en/latest/features.html#transactions).
 
+!!! info
+    The transactions in `lakeFS-spec` are different from the transactions in the [lakeFS-SDK](https://docs.lakefs.io/integrations/python.html#transactions), which were added in `v0.2.0`. 
```

Comment:
```suggestion
    The transactions in `lakeFS-spec` are different from the transactions in the [High-level lakeFS SDK](https://docs.lakefs.io/integrations/python.html#transactions), which were added in `v0.2.0`. 
```

(This makes it more clear that the transactions are from the `lakefs` package, and not from the `lakefs_sdk`.)

Reply:
Agree with @nicholasjng's suggestion, I would just spell it as "high-level SDK" (lowercase).

Also, no backticks around lakeFS-spec and the release version for consistency.